weblate/checks/format.py

```diff
@@ -197,6 +215,10 @@ def c_format_is_position_based(string):
     return "$" not in string and string != "%"
 
 
+def scheme_format_is_position_based(string):
+    return "*" not in string and string != "~"
```

Comment:
Seems like this would be better fit here (guessing based on regexp above, I have no knowledge of scheme):

```suggestion
    return "@*" not in string and string != "~"
```